tests/__init__.py

```diff
@@ -103,6 +103,10 @@ def test_addons_filtered(self):
                 ("bash", "-c", 'addons list -c | grep ,crm,'),
                 # absent_addon is missing and should fail
                 ("bash", "-c", "! addons list -px"),
+                # hr_payroll should have been filtered out
+                ("bash", "-c", "test ! -d auto/addons/hr_payroll"),
```

Comment:
No need for bash here.

Reply:
@Yajo how would it be? `("test", "! -d", "auto/addons/hr_payroll")`?

Reply:
`("test", "!", "-d", "auto/addons/hr_payroll")`

Reply:
Thanks, thought I could only pass three elements :sweat_smile: